src/layers/domain/core/load_questionnaire.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+
+from domain.core.questionnaire_validation_custom_rules import empty_str, url
+from event.json import json_load
+
+from .questionnaire import ALLOWED_QUESTION_TYPES, Questionnaire
+
+PATH_TO_HERE = Path(__file__).parent
+
+VALIDATION_RULE_MAPPING = {
+    "empty_str": empty_str,
+    "url": url,
+}
```

Comment:
sorry realised that we can avoid having to add new rules manually with:

```suggestion
from domain.core import questionnaire_validation_custom_rules
from event.json import json_load

from .questionnaire import ALLOWED_QUESTION_TYPES, Questionnaire

PATH_TO_HERE = Path(__file__).parent
```

and then below do

```python
def process_validation_rules(validation_rules):
    return [questionnaire_validation_custom_rules.__dict__[rule] for rule in validation_rules]
```